apps/central-scan/backend/src/central_scanner_app.test.ts

```diff
@@ -809,17 +809,28 @@ test('auth', async () => {
   });
 
   await apiClient.getAuthStatus();
-  await apiClient.checkPin({ pin: '123456' });
-  await apiClient.logOut();
-
   expect(auth.getAuthStatus).toHaveBeenCalledTimes(1);
   expect(auth.getAuthStatus).toHaveBeenNthCalledWith(1, { electionHash });
+
+  await apiClient.checkPin({ pin: '123456' });
   expect(auth.checkPin).toHaveBeenCalledTimes(1);
   expect(auth.checkPin).toHaveBeenNthCalledWith(
     1,
     { electionHash },
     { pin: '123456' }
   );
+
+  await apiClient.logOut();
   expect(auth.logOut).toHaveBeenCalledTimes(1);
   expect(auth.logOut).toHaveBeenNthCalledWith(1, { electionHash });
+
+  await apiClient.updateSessionExpiry({
+    sessionExpiresAt: new Date().getTime(),
+  });
```

Comment:
Is it expected that real user flows will lead to `updateSessionExpiry` being called by the app after `logOut`? Maybe I missed this in the code earlier but it feels unintuitive; after logging out how can there be a session expiry time (as there is no session)?

Reply:
Ah good catch - I wasn't thinking too much about making this test realistic since the auth instance is mocked out and I'm just verifying that the API call results in the correct auth lib call with the correct input. (I'm relying on `libs/auth` tests for more complete tests of user flows.)

But let me adjust here and elsewhere to make this a little more representative!

Reply:
https://github.com/votingworks/vxsuite/pull/3217/commits/a5534082a3362fb73fb070e9a275941d2a103639

Reply:
Hmm not sure of our testing philosophy - I tend to lean towards making tests as atomic as possible, but that may be the old Mocha way of thinking when you could more easily group like tests under a single `describe` block. Under that philosophy it might make sense to break this larger `auth` test out into smaller atomic tests, but it's definitely low priority since it's an existing pattern to have a single larger test. No strong opinion.

Reply:
That is a great point/suggestion! Updated the app-level auth tests to be more atomic for clarity: https://github.com/votingworks/vxsuite/pull/3217/commits/8f38166cc60cbfce3632edcee34392d9389b5365